docker-compose.yml

```diff
@@ -375,4 +395,5 @@ services:
       - osf_node_modules_vol:/code/node_modules
       - preprints_dist_vol:/preprints
       - registries_dist_vol:/registries
+      - osfstoragecache_vol:/code/website/osfstoragecache
```

Comment:
Coolio, I only left it there so that those working on OSF container would find the files where they expected to.